<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <!--<node pkg="my_package_name" type="my_node_name" name="my_node_name"
 respawn="false" output="screen" args="$(find tracking_control)/config/my_arg_file.yaml"/>-->
 
 <node pkg="tracking_control" type="tracking_control_node" name="TrackingControlNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find tracking_control)/config/config_tracking_control.yaml"/>
</node>
</launch>
